index.html

```diff
@@ -208,6 +210,107 @@ <h3>GetUserMediaSemantics enum</h3>
         </table>
       </div>
     </section>
+    <section id="mediatracksupportedconstraints-dictionary-extensions">
```

Comment:
I think there should be a </section> <section> and a <h2> here to separate the addition of "powerEfficient" from the new picker model section.